### ISSUE TITLE
[Very minor] Missing `lastEntry.setHightlighting(false)` on `recreateLines()` in `GameLog.java`

### ISSUE THREAD

**Issue by Oxtaly** [00-Evan/shattered-pixel-dungeon#1851]
Did not meant to press enter this early ^^', but the title pretty much explains it;

#### TLDR: Very very minor/impossible bug in vanilla game with line drawings's Hightlighting

In the `recreateLines()` method in `GameLog.java`, the entry's Hightlighting does not get set to false, leading to a rare (potentially impossible in the base game) bug where a line containing an underscore gets rendered properly on first pass through (as `setHightlighting(false)` is ran on creation, on line 98) but gets rendered incorrectly when redrawned (like when changing dungeons floors), as the entry's Hightlighting is never set to false.

A simple fix would be to add the line `lastEntry.setHightlighting( false );` at line 138.

Thank you for your time, if this issue is too inconsequential, feel free to close it, and have a good day ^^

**Comment by Oxtaly** [comment]
*Added pictures below that display how the bug looks*
![first_render](https://github.com/user-attachments/assets/1a61db53-0708-41b0-9cc7-da4bcdc295f2)
![recreateLine_render](https://github.com/user-attachments/assets/bc4f8a16-074f-4d4b-a675-d5fa5c832721)

**Comment by Oxtaly** [comment]
Ah, my apologies, I thought that only applied to pull requests, I will refrain from doing so in the future.

Is there a way to talk about potential solutions without creating any copyright issues like avoiding specifying lines of codes, or just simply mentioning an idea would fall under copyright?

Thank you for your time again, and my apologies once more ^^'

**Comment by Oxtaly** [comment]
Fair enough! If I find other bugs I'll simply make a report, have a good day ^^

**Comment by 00-Evan** [comment]
> A poster _explicitly_ declaring a suggestion public domain or CC0 no-attribution should be enough? "have to address an issue in another way" might lead to worse code, and as this case seems limited in (code)scope, there likely won't be many feasible solutions?

Shattered's always been intended as a solo project, so I'd rather just be able to say that all of Shattered's code is by me or Watabou and leave it at that. I'd rather not bother with CLAs or similar when I'm not really interested in pull requests anyway.
